src/api/procedures/initiateCorporateAction.ts

```diff
@@ -130,12 +132,14 @@ export async function prepareInitiateCorporateAction(
   const rawDetails = stringToText(description, context);
   const rawTargets = optionize(targetsToTargetIdentities)(targets, context);
   const rawTax = optionize(percentageToPermill)(defaultTaxWithholding, context);
-  const rawWithholdings = taxWithholdings.map(({ identity, percentage }) =>
-    tuple(
-      stringToIdentityId(signerToString(identity), context),
-      percentageToPermill(percentage, context)
-    )
-  );
+  const rawWithholdings = taxWithholdings
+    ? taxWithholdings.map(({ identity, percentage }) =>
+        tuple(
+          stringToIdentityId(signerToString(identity), context),
+          percentageToPermill(percentage, context)
+        )
+      )
+    : null;
```

Comment:
You can use `taxWithholdings && taxWithholdings.map(...);` instead of the ternary here